packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -0,0 +1,168 @@
+// @flow
+/**
+ * This component turns the given content into an accessible anchor for
+ * positioning and displaying tooltips.
+ */
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import TooltipPortalMounter from "./tooltip-portal-mounter.js";
+
+type Props = {|
+    // A method that renders the content for anchoring the tooltip.
+    // This must return a TooltipPortalMounter component.
+    children: (active: boolean) => React.Element<typeof TooltipPortalMounter>,
+
+    // Callback to be invoked when the anchored content is mounted.
+    // This provides a reference to the anchored content, which can then be
+    // used for calculating tooltip bubble positioning.
+    anchorRef: (?Element) => mixed,
+
+    // When true, the element wrapped by the anchor will be given tabindex=0
+    // to make it keyboard focusable. This value defaults to true. One might set
+    // this to false in circumstances where the wrapped component already can
+    // receive focus or contains an element that can.
+    // Use good judgement when overriding this value, the tooltip content should
+    // be accessible via keyboard in all circumstances where the tooltip would
+    // appear using the mouse, so very those use-cases.
+    forceAnchorFocusivity?: boolean,
+|};
+
+type State = {|
+    active: boolean,
+|};
+
+export default class TooltipAnchor extends React.Component<Props, State> {
+    _weSetFocusivity: ?boolean;
+    _anchorNode: ?Element;
+    _focused: boolean;
+    _hovered: boolean;
+
+    static defaultProps = {
+        forceAnchorFocusivity: true,
+    };
+
+    state = {
+        active: false,
+    };
+
+    componentDidMount() {
+        const anchorNode = ReactDOM.findDOMNode(this);
+
+        // This should never happen, but we have this check here to make flow
+        // happy and ensure that if this does happen, we'll know about it.
+        if (anchorNode instanceof Text) {
+            throw new Error(
+                "TooltipAnchor must be applied to an Element. Text content is not supported.",
+            );
+        }
+
+        this._anchorNode = anchorNode;
+        this._updateFocusivity();
+        if (anchorNode) {
+            // TODO(somewhatabstract): Detect ESC key to deactivate
+            // TODO(somewhatabstract): Work out how to allow pointer to go over
+            // the tooltip content to keep it active.
```

Comment:
This seems tricky.  How important is this compared to other a11y considerations?

Reply:
Yeah, I am not going to tackle the pointer part just yet. That might have to go into the backlog.

Like any a11y considerations, sometimes we can't do them all. We have to do the best we can. I'd rather think on it a little more before we commit to a solution. Especially since this requires pointer events and that breaks our "fixed/absolute" obscurement checks a little (as they are currently implemented).